src/stylesheets/components/_footer.scss

```diff
@@ -355,8 +355,10 @@ li.usa-footer-primary-content {
       }
     }
 
-    li {
-      line-height: 2em;
+    li:not(.usa-footer-primary-link) {
+      line-height: 1.35;
```

Comment:
Can we use one of our line height variables for this? https://github.com/18F/web-design-standards/blob/develop/src/stylesheets/core/_variables.scss#L24-L26

Reply:
When I use `$heading-line-height` or `1.3`, it seems to have the same effect / doesn't change the size, so maybe we can stick with that?

Reply:
I'll use `$heading-line-height` here. Forgot those existed.